src/components/blog/details/index.tsx

```diff
@@ -45,12 +46,15 @@ export const BlogDetails = (): JSX.Element => {
                         <div className="card-authors-image">
                             <Link to={`/user/${post.author.username}`}><img className="circle" src={post.author.gravatar_url} /></Link>
                         </div>
-                        <div className="card-authors-text">
+                        <div className="card-authors-text" style={{ display: "flex", gap: "0.25em" }}>
```

Comment:
hmm, maybe avoid inline styling for the sake of clarity

Reply:
> hmm, maybe avoid inline styling for the sake of clarity

oh yep forgot to remove that thanks for noticing